### ISSUE TITLE
QB and sound interaction with the user

### ISSUE THREAD

**Issue by meteos77** [textbrowser/biblioteq#258]
today I tested the BA/Discover feature : it works very well 

I have a suggestion for improvement : Can BQ make a sound when it can't find the document. This would avoid having to look at the screen to see if BQ has the document in its database.

**Comment by textbrowser** [comment]
You'd like a ting on success and a bong on failure. Or something. I could introduce blue or green text if that helps. Media sound files are nice but require audio cards.

**Comment by meteos77** [comment]
last week I inventoried 250 books
237 were present so the interface answered "book".
but I had to look at the screen 250 times to see if the field answered book.

A sound for a failure would have allowed me to concentrate on the documents and the computer beeps for a long time?
just on the error not on the successes because it will become tiresome for the ears :-)

**Comment by textbrowser** [comment]
There's nothing standard about a beep.

**Comment by textbrowser** [comment]
So again, from previous discussions, proprietary software bundled with hardware includes audios because they're manufactured with such features. They're not features requested and added 20 years after the software is developed.

**Comment by textbrowser** [comment]
A beep may indicate a problem. So nothing standard about that too. I'd perform the scanning and then visually inspect the results. It's like a spell checker. You have to spell check. Audio will not help.

**Comment by textbrowser** [comment]
Yeah, you're requesting a feature that is not standard, is ambiguous, may be annoying if it's not optional (so that's another hit against this request because it requires yet another option), cannot be guaranteed to work, can cause a system to fail, etc.).

https://www.dell.com/community/Laptops-General-Read-Only/Beep-then-crash/td-p/1390807

**Comment by meteos77** [comment]
you need two "special BQ" sounds

if BQ does not know the scanned document, it emits the sound "not found
if BQ knows the scanned document, it emits the "find in database" sound

**Comment by textbrowser** [comment]
I'm offering an improvement and you're arguing against it. OK!

**Comment by meteos77** [comment]
I did not understand anything then; I am for the improvement

**Comment by meteos77** [comment]
you know that I am for all improvements :-)

**Comment by textbrowser** [comment]
You have an Android? Smoke has a generic pleasant sound when a participant receives a message. So, something like that. [Standard] failure and [standard] success audio files. I also added something like this in Spot-On.

**Comment by meteos77** [comment]
no recent android 4.2 or 4.4

**Comment by textbrowser** [comment]
I prefer laptops and then tablets. Tablets before small devices. Then there are portable Pis. Keyboards, monitors.

**Comment by meteos77** [comment]
branches_trunk_Sounds_send.wav and branches_trunk_Sounds_receive.wav are cool

the other 2 more aggressive

**Comment by meteos77** [comment]
tablets are good for people without computers

e.g.: my mother can look at her photos, play games (card, puzzle ...)

**Comment by meteos77** [comment]
smartphone : having a computer in the pocket is cool, we have dreamed of it in our youth

**Comment by textbrowser** [comment]
OK, adding an option now. :) A sound file is played if an item is discovered.

**Comment by meteos77** [comment]
as usual great

can we have a different sound in case of non discovery (red)
